src/api.py

```
@@ -0,0 +1,390 @@
+"""
+FastAPI REST API interface for the Agent-News system
+Provides external access to news, sentiment analysis, and trading signals
+"""
+
+from fastapi import FastAPI, HTTPException, Query, BackgroundTasks
+from fastapi.responses import JSONResponse
```

Ruff (0.11.9)

`fastapi.responses.JSONResponse` imported but unused

Remove unused import: `fastapi.responses.JSONResponse`

(F401)